src/main/resources/application.yaml

```diff
@@ -4,6 +4,10 @@ server:
 spring:
   application:
     name: rollup-unified-metrics
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
Do we know if this is actually needed?

Whenever I've received the warning suggesting I could enable this I've always been able to fix things a different way.  It feels like this is one of those settings that hides other problem rather than fixes them.

Reply:
It seems to start up ok without it.  But I'm always a bit hesitant about making non urgent changes to things I don't understand well.  I'm going to hold off for now until I understand the app better.